DOCSTYLE.md

```diff
@@ -0,0 +1,45 @@
+# Documentation style guidance
+
+We've defined some basic patterns to bring our documentation towards a more consistent, readable style.
+
+- We are committed to inclusive communication
+- We use British English spelling (e.g., "organisation" not "organization", "centre" not "center")
```

Comment:
why?

Reply:
Mastodon is a European organisation and we have specifically chosen to use British English as an expression of our identity.